django/researchdata/migrations/0001_initial.py

```diff
@@ -36,6 +38,8 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=255, unique=True)),
+                ('location', models.CharField(blank=True, max_length=255, null=True, verbose_name='location (search)')),
+                ('location_coordinates', location_field.models.plain.PlainLocationField(blank=True, max_length=63, null=True, verbose_name='location (coordinates)')),
```

Comment:
I presume that there is a specific reason for `max_length` being set to 63.

Reply:
I copied that line from their documentation so perhaps that's as long as coordinates can be... but just in case, I'll change it to 255 to allow some breathing room

Reply:
oh wait, that's the automatically generated migration file. so no need to review that. the max length will have been set by the PlainLocationField